bootstrapvz/plugins/expand_root/README.rst

```diff
@@ -4,7 +4,7 @@ Expand Root
 This plugin adds support to expand the root partition and filesystem dynamically on boot. It adds a shell script to call growpart and the proper filesystem expansion tool for a given device, partition, and filesystem. The growpart script is part of the cloud-guest-utils package in stretch and jessie-backports. The version of this script in jessie is broken in several ways and so this plugin installs the version from jessie-backports which works correctly. This plugin should not be used in conjunction with common.tasks.initd.AddExpandRoot and common.tasks.initd.AdjustExpandRootScript. It is meant to replace the existing internal common version of expand-root.
 
 Settings
---------
+~~~~~~~~
```

Comment:
Yes, I did. You can also compare the headline level of the Settings subsubsection in other plugin README.rst files. They all have `~~~`. See also the currently [generated docs](http://bootstrap-vz.readthedocs.io/en/master/plugins/index.html)—the two files I am fixing now have their Settings subsubsections as subsections in the list on the left.